gatsby-config.js

```diff
@@ -26,7 +31,8 @@ module.exports = {
     siteUrl: 'https://yummy.ertrzyiks.me',
     author: 'Author Name'
   },
-  plugins: sourcePlugins.concat([
+  plugins: localPlugins.concat(sourcePlugins).concat([
+    `gatsby-yummy-search-data`,
```

Comment:
Has this plugin not already been loaded as part of `localPlugins` (line 5)?

Reply:
Nice catch!